DocumentedExamples/Querying_Scalar_Quantities_and_Annually_Averaged_Timeseries.ipynb

```diff
@@ -0,0 +1,2091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">results = {}
variable = "total_ocean_salt"
for experiment in experiments:
  try:
    results[experiment] = cc.querying.getvar(experiment, variable, session, ncfile="ocean_scalar.nc")
  except cc.querying.VariableNotFoundError:
    print(f"No {variable} in {experiment}")
</pre><p> </p><p>I think this is cleaner.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/247/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>